jr-objects/src/test/java/com/fasterxml/jackson/jr/GroovyTest.groovy

```diff
@@ -0,0 +1,58 @@
+package com.fasterxml.jackson.jr
+
+import com.fasterxml.jackson.jr.ob.JSON
+import com.fasterxml.jackson.jr.ob.TestBase
+
+class GroovyTest extends TestBase {
+
+    void testSimpleObject() throws Exception {
+        var data = JSON.std.asString(new MyClass())
+        var expected = """{"AAAAA_A_Field_Starting_With_Two_Capital_Letters":"XYZ","aDouble":0.0,"aPublicInitializedInteger":56,"aPublicInitializedIntegerObject":1516,"aPublicUninitializedInteger":0,"anInitializedIntegerObject":1112,"anInitializedPublicString":"stringData","anInitializedString":"ABC","anInteger":0,"anIntegerWithValue":12}"""
```

Comment:
Was about to ask about `var` and triple quotes wrt JDK 8 before realizing this is Groovy :)

Reply:
One question tho: how does this test get run? It doesn't look like it runs with `./mvnw clean test` from main level...

Reply:
As per 

https://ppettitau.wordpress.com/2013/04/04/running-groovy-junit-tests-with-surefire-in-a-javamaven-project/

that (moving test) is one of 3 things needed, fwtw.